### DISCUSSION TITLE
Is `--shellcode` required, or what am I missing?

### DISCUSSION THREAD

**Discussion by gaggle** [pkgxdev/pkgx#730] [Help] [answered]
I may be a real dummy here, but when I follow the installation guides I don't end up with tea that works:

```sh
$ brew install teaxyz/pkgs/tea-cli
...
$ tea +node
...
$ which node
node not found
$ which tea
/usr/local/bin/tea
$ tea --version
tea 1.0.0-alpha.3
```

But when I `eval "$(tea --shellcode)"` then everything works as advertised and it's **glorious**:

```sh
$ eval "$(tea --shellcode)"
$ tea +node
$ which node
~/.tea/nodejs.org/v20.6.1/bin/node
```

But is `shellcode` required? If it is, shouldn't that be on the website's front-page and in the docs? If it isn't, then.. what did I miss?

**Comment by mxcl** [comment, marked as answer]
This is an alpha issue, a soon release will tell you you need to `tea integrate` when you type such commands before they will work. This is documented but it's easy to miss.

**Reply by gaggle** [reply]
Perfect, thanks for the clarification.

**Reply by mxcl** [reply]
alpha.4 adds the text that recommends the user run `tea integrate`.
